README.md

```diff
@@ -33,5 +33,5 @@ It is a general purpose statically typed, with dynamic types, easy to learn and
 #### Manuals:
 - [User Manual](../userman/html/index.html)
 - [Reference Manual](../refman/html/index.html)
-- [Bill_Primer](../langman/html/index.html)
+- [Bill_Primer](langman/html/index.html)
```

Comment:
In README.md, you want: https://jsmith1024.github.io/bill/langman/html/index.html

Note: README.md as rendered on github.com needs to link to the github page explicitly.  You might also update "Manuals:" to "Current Manuals:" or such.
This is different than what you want in the .dox pages which, when rendered use nice relative addresses.